mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulContestsScreen.kt

```diff
@@ -0,0 +1,60 @@
+package ch.epfl.sdp.mobile.state
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import ch.epfl.sdp.mobile.ui.tournaments.*
+import ch.epfl.sdp.mobile.ui.tournaments.Status.*
+import ch.epfl.sdp.mobile.ui.tournaments.Status.ContestStatus.*
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.days
+import kotlin.time.Duration.Companion.hours
+
+/** An adapter that is of type [Contest] and contains the uid. */
+data class ContestAdapter(
+    val uid: String,
+    override val name: String,
+    override val creationTime: Duration,
+    override val personStatus: BadgeType,
+    override val status: ContestStatus
+) : Contest {}
+
+/**
+ * An implementation of the [ContestScreenState] that performs a given profile's [Contest] requests.
+ */
+class TournamentScreenState() : ContestScreenState<ContestAdapter> {
+
+  // override var contests by mutableStateOf(emptyList<Contest>())
+  // private set
+  override val contests =
+      listOf(
+          createContest("1", "EPFL Grand Prix", 1.days, ONGOING, BadgeType.Admin),
+          createContest("2", "Pawn Party", 2.days, DONE, BadgeType.Participant),
+          createContest("3", "Never gonna chess", 3.hours, ONGOING, BadgeType.Join))
```

Comment:
When you create a dummy data like this, add To-do keyword to notify to other teammate that it is a temporary value